Tests/AsyncHTTPClientTests/HTTP2ClientTests.swift

```diff
@@ -67,7 +82,7 @@ class HTTP2ClientTests: XCTestCase {
     func testConcurrentRequestsFromDifferentThreads() {
         let bin = HTTPBin(.http2(compress: false))
         defer { XCTAssertNoThrow(try bin.shutdown()) }
-        let client = self.makeDefaultHTTPClient()
+        let client = self.makeClientWithActiveHTTP2Connection(to: bin)
```

Comment:
why do we need a client with an active connection?

Reply:
good catch, we don't.